source/mainnet/net/guides/run-local-chain.rst

```diff
@@ -0,0 +1,185 @@
+
+Next, generate the genesis data:
+
+.. code-block:: console
+
+    $ ./target/release/genesis-creator generate --config ./examples/genesis5.toml
+    Deleting any existing directories.
+    Account keys will be generated in ./accounts
+    Chain update keys will be generated in ./update-keys
+    Identity providers will be generated in ./idps
+    Anonymity revokers will be generated in ./ars
+    Baker keys will be generated in ./bakers
+    Cryptographic parameter will be generated in ./global
+    The genesis data will be stored in ./genesis.dat
+    The genesis hash will be written to ./genesis_hash
+    There are 2 accounts in genesis, 1 of which are bakers.
+    Genesis time is set to 2023-05-22 15:08:19.803 UTC.
+    Average block time is set to 5000ms.
+    DONE
+
+In particular, the files of interest to you are ``./genesis.dat`` containing the genesis block data and ``./bakers/baker-0-credentials.json`` containing the credentials of the single baker account that was created.
+
+
+Running the chain
+=================
+
+Now run the chain by starting a single baker node. The node expects the ``genesis.dat`` to reside in a configuration directory, so first create a working directory for the node data and configuration and copy ``genesis.dat`` to it:
+
+.. code-block:: console
+
+    mkdir localchain-node-0
+    cp ./genesis.dat localchain-node-0/
+
+Now run the baker node as follows:
+
+.. code-block:: console
+
+    concordium-node \
+      --no-bootstrap= \
+      --grpc2-listen-addr 127.0.0.1 \
+      --grpc2-listen-port 20001 \
+      --data-dir localchain-node-0 \
+      --config-dir localchain-node-0 \
+      --baker-credentials-file bakers/baker-0-credentials.json
+
+The ``--no-bootstrap`` option lets the node know not to connect to a bootstrapper node for retrieving peers since no peers are in this network so this is not relevant. The ``--grpc2-listen-port`` specifies the port to listen on for Node GRPC V2 API handshakes. We will use this interface to communicate with the node. The ``--data-dir`` and ``--config-dir`` specifies the working directories of the node instance, where its state and configuration is stored. The ``--baker-credentials-file`` instructs the node to run as the baker specified in the supplied credentials file, in this case your generated baker credentials output from the ``genesis-creator`` tool.
+
+.. Note::
+
+    If more baker credentials are generated, more bakers can be spun up by replacing the arguments specified by the ``--baker-credentials-file``, ``--config-dir`` and ``--data-dir`` options accordingly. If there is no bootstrapper node, it will have to be instructed to manually connect to one another by specifying the IP address and port of the other node(s) using the ``--connect-to`` option. Note that nodes running on the same network interfaces must specify a disjoint set of ports.
```

Comment:
Maybe just get rid of this note?

Reply:
I think it's ok to have. I'm not sure why you are mentioning sets of ports though. You are only specifying one for the grpc. 

Maybe you wanted to include the `listen-port`  for specifying the TCP port that the node uses to receive messages on from other peers.

Reply:
I mention it since that it would fail if we two node instances attempt to listen on the same network socket. But the phrasing is admittedly incorrect and I will change it :-) thanks!

Reply:
I think the note is useful.

But add that the different nodes must use different config and data dirs. That's a bit implicit now.

Reply:
Added.